server/config/tagType/tagConfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "REJECTED_BY_PSU": [
+    {
+      "tagName": "REJECTED BY PSU",
+      "tagLabel": "govuk-tag govuk-tag--red"
+    }
+  ],
+  "HS": [
+    {
+      "tagName" : "HS",
+      "tagLabel": "govuk-tag govuk-!-margin-right-1"
```

Comment:
`govuk-!-margin-right-1`

Feels a little odd to have that on just this tag? I haven't pulled and checked the branch but this might be worth doing in the Tag component if theres more than one? Or we could bake some CSS in all apart from N-th last?

Happy to jump on a call for this 🙂